tmail-backend/apps/distributed/src/test/resources/pop3server.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!-- Read https://james.apache.org/server/config-pop3.html for further details -->
+
+<pop3servers>
+    <pop3server enabled="false">
```

Comment:
enabled = false?

Reply:
yes, jut for CI pass
FOR [MU], just changing enabled=true

Reply:
> yes, jut for CI pass

Does it mean we need to add a dummy `pop3server.xml` file for every other deployments?

Reply:
hmm, maybe yep, but wait to CI result :P

Reply:
Better (I tried):

```
<pop3servers>
</pop3servers>
```

Likely there was a default POP3 configuration somewhere that tried to open directly for example the 995 port.

Reply:
I personnally like the module chooser idea, if we don't need pop3 for a deployment we just dont load it.

Reply:
Do we need contract tests for the POP3 Tmail server? 
It seems more complicated than the first commit. `Add support for POP3 in Twake mail`